lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -308,77 +279,8 @@ class StandardAccountTest {
         }
     }
 
-    @Test
-    fun `estimate message fee`() {
-        // Note to future developers experiencing failures in this test.
-        // Compiled contract format sometimes changes, this causes changes in the class hash.
-        // If this test starts randomly falling, try recalculating class hash.
-
-        // TODO: Migrate l1l2 contract to Cairo 1.
-        val l1l2ContractCode = Path.of("src/test/resources/contracts_v0/target/release/l1l2.json").readText()
-        val l1l2ContractDefinition = Cairo0ContractDefinition(l1l2ContractCode)
```

Comment:
We still want to test `estimateMessageFee`. 

Instead of removing this test, we should update it to use Cairo 1 contract, you can checkout the [one](https://github.com/software-mansion/starknet.swift/blob/main/Tests/StarknetTests/Resources/Contracts/src/l1_l2.cairo) used in starknet.swift. You might also have slightly update the test similarly to https://github.com/software-mansion/starknet.swift/blob/5aed9f911e5180f265f879cdc3a9ab62882ad97c/Tests/StarknetTests/Providers/ProviderTests.swift#L290

Reply:
`estimateMessageFee` has been updated, L1 L2 messaging demo contract has been bumped to Cairo 2 as we discussed.